src/components/player.js

```diff
@@ -99,8 +99,11 @@ export default class Player extends BaseObject {
    * define a poster by adding its address `poster: 'http://url/img.png'`. It will appear after video embed, disappear on play and go back when user stops the video.
    * @param {String} [options.playbackNotSupportedMessage]
    * define a custom message to be displayed when a playback is not supported.
+   * @param {Object} readyCallback (optional)
+   * If provided this will be called when the player has finished loading (just before the READY event is fired).
+   * It will be provided the player instance as the first argument.
    */
-  constructor(options) {
+  constructor(options, readyCallback) {
```

Comment:
We can discuss that with @clappr/core :) with that said I think it'd be nicer if we provide it as an option, then we can extend to a set of most used events (onPause, onPlay, onStop, onError(this one might be pretty useful) ...):

```javascript
new Clappr.Player({
events: {
  onReady: "..",
  onPlay: "..."
}
})
```